src/gui/widgets/text_box_base.hpp

```diff
@@ -155,6 +155,10 @@ class text_box_base : public styled_widget
 	 */
 	void set_selection(std::size_t start, int length);
 
+    /*
```

Comment:
Please start the doc with `/**`. Also, this is indented with spaces in a file that's otherwise indented with tabs.

Reply:
Updated.